core/metacling/src/TClingCallbacks.cxx

```diff
@@ -921,7 +921,8 @@ void TClingCallbacks::UnlockCompilationDuringUserCodeExecution(void *StateInfo)
 
 static bool shouldIgnore(llvm::StringRef FileName) {
    llvm::StringRef fileStem = llvm::sys::path::stem(FileName);
-   return fileStem.startswith("libNew");
+   return fileStem.startswith("libNew") ||
+      fileStem.startswith("libcppyy_backend");
```

Comment:
@etejedor, ROOT has a dynamic linker (dyld) which can scan symbols from a given library. In a few cases (especially when explicit linking is disabled) before loading a library, ROOT will ask what are the dependent libraries to load first.

With python we have a new challenge: we should not resolve symbols from the libcppyy*{2_7,3_6} because we may break the original intent. For example, dyld can resolve a symbol from `libcppyy_backend2_7` but then libROOTTPython to decide loading libcppyy_backend3.

This logic here filters that out but is only `libcppyy_backend` intended to be accessed via `libROOTTPython` or should I exclude `libROOTPythonizations`, too?